packages/runtime/src/runtimes/react/controls/rich-text-v2/rich-text-v2.tsx

```diff
@@ -130,17 +105,34 @@ export function useRichTextV2(data: RichTextV2ControlData, control: RichTextV2Co
 
   const handleKeyDown = useCallback(
     (e: KeyboardEvent) => {
+      if (editMode === BuilderEditMode.CONTENT) e.stopPropagation()
       if (isHotkey('mod+shift+z', e)) return HistoryEditor.redo(editor)
       if (isHotkey('mod+z', e)) return HistoryEditor.undo(editor)
       if (isHotkey('escape')(e)) {
         isPreservingFocus.current = false
         ReactEditor.blur(editor)
         control?.switchToBuildMode()
       }
+
+      plugins.forEach(plugin => plugin?.onKeyDown?.(e))
     },
     [control, editor],
   )
 
+  const handleKeyUp = useCallback(
+    (e: KeyboardEvent) => {
+      if (editMode === BuilderEditMode.CONTENT) e.preventDefault()
```

Comment:
Repro'd this in codesandbox and it is the default behavior for the events to propagate out to elements outside of content editable.

Reply:
Event propagation and default behavior are separate things (i.e., `event.stopPropagation()` and `event.preventDefault()`) so I'm not understanding what you mean in this comment